src/main/resources/db/changelog/changes/v1.0.4/sql/derived-tables/create-view-purchase-order-lines.sql

```diff
@@ -0,0 +1,56 @@
+	concat((user_details2.jsonb -> 'personal'::text) ->> 'lastName'::text, ', ',
+              (user_details2.jsonb -> 'personal'::text) ->> 'firstName'::text)               AS purchase_order_line_created_by,
+    (po.jsonb -> 'metadata'::text) ->> 'updatedByUserId'::text AS purchase_order_line_updater_id,
+	concat((user_details3.jsonb -> 'personal'::text) ->> 'lastName'::text, ', ',
+              (user_details3.jsonb -> 'personal'::text) ->> 'firstName'::text)               AS purchase_order_line_updated_by,
+    po.jsonb ->> 'description'::text AS purchase_order_line_description,
+    (po.jsonb -> 'cost'::text) ->> 'poLineEstimatedPrice'::text AS purchase_order_line_estimated_price,
+    (po.jsonb -> 'cost'::text) ->> 'exchangeRate'::text AS purchase_order_line_exchange_rate,
+    COALESCE(json_array_elements.value ->> 'value'::text, NULL::text) AS fund_distribution_value,
+    COALESCE(json_array_elements.value ->> 'code'::text, NULL::text) AS fund_distribution_code,
+    COALESCE(json_array_elements.value ->> 'distributionType'::text, NULL::text) AS fund_distribution_type,
+    COALESCE(json_array_elements.value ->> 'fundId'::text, NULL::text) AS fund_id,
+    COALESCE(json_array_elements.value ->> 'encumbrance'::text, NULL::text) AS fund_distribution_encumbrance,
+    po.jsonb ->> 'purchaseOrderId'::text AS purchase_order_id,
+    o.jsonb ->> 'approved'::text AS  purchase_order_approved,
+    o.jsonb ->> 'orderType'::text AS  purchase_order_type,
+    o.jsonb ->> 'poNumber'::text AS po_number,
+    o.jsonb ->> 'workflowStatus'::text AS  purchase_order_workflow_status,
+    (o.jsonb -> 'metadata'::text) ->> 'createdDate'::text AS  purchase_order_created_date,
+    (o.jsonb -> 'metadata'::text) ->> 'createdByUserId'::text AS  purchase_order_created_by_id,
+	concat((user_details.jsonb -> 'personal'::text) ->> 'lastName'::text, ', ',
+              (user_details.jsonb -> 'personal'::text) ->> 'firstName'::text) AS  purchase_order_created_by,
+    (o.jsonb -> 'metadata'::text) ->> 'updatedDate'::text AS purchase_order_updated_date,
+	concat((user_details1.jsonb -> 'personal'::text) ->> 'lastName'::text, ', ',
+              (user_details1.jsonb -> 'personal'::text) ->> 'firstName'::text)     AS  purchase_order_updated_by,
+    (o.jsonb -> 'metadata'::text) ->> 'updatedByUserId'::text AS purchase_order_updated_by_id,
+    o.jsonb ->> 'vendor'::text AS vendor_id,
+    organization_details.jsonb ->> 'code'::text AS vendor_code,
+    organization_details.jsonb ->> 'name'::text AS vendor_name,
+    o.jsonb ->> 'assignedTo'::text AS purchase_order_assigned_to_id,
+    concat((user_details4.jsonb -> 'personal'::text) ->> 'lastName'::text, ', ',
+              (user_details4.jsonb -> 'personal'::text) ->> 'firstName'::text) AS purchase_order_assigned_to,
+    jsonb_array_elements(o.jsonb -> 'notes'::text)::text AS purchase_order_notes,
+    ( SELECT array_to_string(array_agg(dept_id.value),',') AS array_agg
```

Comment:
It doesn't look like this works. For example, PO cd703193-054d-4ddc-a3f8-da9f25e8647e in corsair-frm has an acqUnitId, but that field shows up empty in the view that this creates. 

Looking at the fields in the purchase_order schema, it looks like the acqUnitIds need to come from the purchase_order, not from the POL.